cmake/macros/macro_deal_ii_add_library.cmake

```diff
@@ -52,7 +52,7 @@ MACRO(DEAL_II_ADD_LIBRARY _library)
     # Cuda specific target setup:
     #
     IF(DEAL_II_WITH_CUDA)
-      CUDA_WRAP_SRCS(${_library}.${_build_lowercase}
+      CUDA_WRAP_SRCS(${_library}_${_build_lowercase}
```

Comment:
The "automagic" target dependencies in ```CUDA_WRAP_SRCS``` don't work anyway (this was this funny bug we hunted down for 2 weeks if I recall correctly).